packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -354,14 +354,26 @@ export const UIModeView: React.FC<{}> = ({
     return () => disposable.dispose();
   }, [runTests, testServerConnection, watchAll, watchedTreeIds, teleSuiteUpdater, projectFilters]);
 
+  const hideOutput = React.useCallback(() => setIsShowingOutput(false), [setIsShowingOutput]);
+  const showOutput = React.useCallback(() => {
+    setIsShowingOutput(true);
+    setOutputContainsError(false);
+  }, [setIsShowingOutput, setOutputContainsError]);
```

Comment:
Both `setIsShowingOutput` and `setOutputContainsError` are React `useState` setters, and thus are static. `rules-of-hooks` will not flag you if you do not include them in the dependency arrays, so I would recommend that you do not include them, as they are unnecessary.